fabric/main.py

```diff
@@ -125,6 +126,7 @@ def is_classic_task(tup):
             callable(func)
             and (func not in _internals)
             and not name.startswith('_')
+            and not (inspect.isclass(func) and issubclass(func, Exception))
```

Comment:
Fair enough, I do agree that some crazy person could be using `__init__` as a task and I agree this seems less likely to break for anyone.